server/models/historical_meteorological.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Column, Float, Integer, String
+
+from server.db.base_class import Base
+
+
+class Historical_Meteorological(Base):
+    __tablename__ = "Historical Meteorological Data"
+
+    id = Column(Integer, index=True, nullable=False, primary_key=True)
+    station_id = Column(String(10), index=True, nullable=False, unique=False)
```

Comment:
If you were to diagram out the tables that we're creating here, how would it look like? What relationships are we trying to form and why?

To complete the relationship implementation, check out [this part](https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html) from `sqlalchemy`'s docs for guidance with the ORM's syntax.

Reply:
Forgot to label station_id `foreign_key=True`

Reply:
What about defining the relationship attribute? Is that something we'd want to use?

Check out the [coastlines model](https://github.com/clairBuoyant/server/blob/main/server/models/coastline.py#L30) to see how buoy data is accessible via our ORM's [relationship API](https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship).

Reply:
Already did that while building in ForeignKeys. Just didn't mention it because it should go without saying lol.